tests/bsim/bluetooth/host/adv/periodic/prj_coded.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_BT=y
+CONFIG_BT_DEVICE_NAME="test_per_adv"
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
```

Comment:
I might be missing something here, but where do we need the peripheral and central role support?

Don't we just need the broadcaster and observer roles?

Reply:
You are right, it is strictly speaking not needed